sympy/calculus/singularities.py

```diff
@@ -238,15 +206,13 @@ def is_monotonic(f, interval=S.Reals, symbol=None):
     True
 
     """
-    from sympy.core.logic import fuzzy_or
-    f = sympify(f)
-    free_sym = f.free_symbols
-
-    if symbol is None and len(free_sym) > 1:
-        raise NotImplementedError('is_monotonic has not yet been '
-                                'for all multivariate expressions')
+    expression = sympify(expression)
 
-    inc = is_increasing(f, interval, symbol)
-    dec = is_decreasing(f, interval, symbol)
+    if symbol is None and len(expression.free_symbols) > 1:
+        raise NotImplementedError(
+            'is_monotonic has not yet been implemented'
+            ' for all multivariate expressions.'
+        )
 
-    return fuzzy_or([inc, dec])
```

Comment:
`is_subset` is decisive (will either return `True` or `False` or raise an exception, so (as I can see) there is no need to use `fuzzy_or` here.